Ginger/GingerUtils/OS/OperatingSystemBase.cs

```diff
@@ -75,5 +76,32 @@ private static OperatingSystemBase GetOperatingSystem()
             }
         }
 
+        /// <summary>
+        /// Convert Solution Relative Path to Full path
+        /// </summary>
+        /// <param name="relativePath">Path like "~\Documents\Scripts\aa.vbs"</param>
+        /// <returns></returns>
+        public string ConvertSolutionRelativePath(string relativePath, string mSolutionFolderPath)
+        {
+            if (String.IsNullOrWhiteSpace(relativePath))
+            {
+                return relativePath;
+            }
+            try
+            {
+                if (relativePath.TrimStart().StartsWith("~"))
+                {
+                    string fullPath = relativePath.TrimStart(new char[] { '~', '\\', '/' });
+                    fullPath = Path.Combine(mSolutionFolderPath, fullPath);
+                    return CurrentOperatingSystem.AdjustFilePath(fullPath);
+                }
+            }
+            catch (Exception ex)
+            {
```

Comment:
Avoid empty catch, please add logs

Reply:
We do have access to Reporter in this project so removed try catch so that exception will be sent to the caller and can be logged there.